tests/sharded_jit_test.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from functools import partial
+from unittest import SkipTest
+
+import numpy as onp
+from absl.testing import absltest
+
+import jax
+import jax.numpy as np
+from jax import test_util as jtu
+from jax.interpreters import pxla
+from jax.interpreters.sharded_jit import sharded_jit
+from jax.interpreters.sharded_jit import PartitionSpec as P
+
+from jax.config import config
+config.parse_flags_with_absl()
+
+
+class ShardedJitTest(jtu.JaxTestCase):
+
+  def testBasic(self):
+    if jtu.device_under_test() != "tpu" or jax.device_count() < 2:
```

Comment:
Maybe move this into `def setUp(self)` if it is common for all tests?

Reply:
Good idea, done.